### PR TITLE
YONK-950: Hotspots feedback is showing properly on Studio

### PR DESCRIPTION
**Pull request by aamishbaloch**
Changes include Hotspots feedback design fix for image explorer in studio. 
Ticket Link: [YONK-950](https://openedx.atlassian.net/browse/YONK-950)

@pomegranited Can you please review this and add this commit to the branch which you guys use to add this xblock in platform (hint: requirements.txt).

### CODE REVIEW

**Comment by pomegranited** [comment]
Sure thing @aamishbaloch -- I'll do this first thing next week.

File: image_explorer/public/css/image_explorer.css [hunk 1]
```diff
@@ -151,3 +151,7 @@ div.image-explorer-hotspot .image-explorer-hotspot-reveal .image-explorer-hotspo
     display: inline-block;
     width: 90%;
 }
+
+.xblock-render{
+    overflow: unset !important;
```
Comment (pomegranited) [inline image_explorer/public/css/image_explorer.css]: Need to use `overflow: visible !important` for this to work in IE11.  `unset` doesn't seem to be supported.

**Comment by aamishbaloch** [comment]
@pomegranited Done with the changes. Can you review the changes?

**Comment by aamishbaloch** [comment]
@pomegranited Thanks

**Pull request merged**